### PR TITLE
feat: use new top level wash commands

### PR DESCRIPTION
**Pull request by vados-cosmonic**
## Feature or Problem

Add top level `wash` commands to the documentation, replacing relevant existing uses of `wash ctl <x>`

See also: https://github.com/wasmCloud/wash/issues/538

example of replacement

## Related Issues

https://github.com/wasmCloud/wash/issues/538

## Testing

Built on platform(s)
- [ ] x86_64-linux
- [ ] aarch64-linux
- [ ] x86_64-darwin
- [ ] aarch64-darwin
- [ ] x86_64-windows

Tested on platform(s)
- [ ] x86_64-linux
- [ ] aarch64-linux
- [ ] x86_64-darwin
- [ ] aarch64-darwin
- [ ] x86_64-windows

### CODE REVIEW

**Review by brooksmtownsend — CHANGES_REQUESTED** [review]
It's surprising how much cleaner the commands look, especially without `ctl` and things.

Only blocking on us releasing wash 0.18

File: blog/better-together-building-efficient-microservices-in-kubernetes-with-webassembly.md [hunk 1]
```diff
@@ -100,12 +100,16 @@ We’re also using a handy wasmCloud Actor written by Taylor Thomas at Cosmonic,
...
-wash ctl start actor wasmcloud.azurecr.io/service_applier:0.3.0
-wash ctl start actor wasmcloud.azurecr.io/echo:0.3.8
+wash start provider wasmcloud.azurecr.io/applier:0.3.0
+wash start provider wasmcloud.azurecr.io/nats_messaging:0.17.0
+wash start provider wasmcloud.azurecr.io/httpserver:0.17.0
+wash start actor wasmcloud.azurecr.io/service_applier:0.3.0
+wash start actor wasmcloud.azurecr.io/echo:0.3.8
 ```
+:::info
+Previous guides used `wash ctl start` which is deprecated and will be replaced by `wash start` in a future version
```
Comment (connorsmith256) [inline blog/better-together-building-efficient-microservices-in-kubernetes-with-webassembly.md]: Same comment everywhere. The "will be replaced by wash start" kinda sounds like `wash start` isn't available yet, but the code snippets show that it is

```suggestion
Previous guides used `wash ctl start`, which is now deprecated and will be removed in a future version
```

**Review by connorsmith256 — CHANGES_REQUESTED** [review]
Looks like there are still some references to `wash reg` and `wash claims` that should be updated now too

File: blog/better-together-building-efficient-microservices-in-kubernetes-with-webassembly.md (hunk 1 above)
Comment (vados-cosmonic) [reply blog/better-together-building-efficient-microservices-in-kubernetes-with-webassembly.md]: - [x] fix wording on deprecation

**Comment by vados-cosmonic** [comment]
Oh sorry @brooksmtownsend re-requested review from you when we're still waiting on wash `0.18`.

@connorsmith256 please take a look! I fixed the `wash reg` usages

**Review by connorsmith256 — APPROVED** [review]
Woohoo!

**Comment by connorsmith256** [comment]
(Still waiting to merge until after v0.18 is released)

**Review by brooksmtownsend — APPROVED** [review]
good to go!

**Pull request merged**